pkg/command/lint/sarif_format.go

```diff
@@ -0,0 +1,56 @@
+package lint
+
+import (
+	"io"
+
+	"github.com/owenrumney/go-sarif/sarif"
+	"github.com/pkg/errors"
+	"golang.stackrox.io/kube-linter/pkg/run"
+)
+
+// FormatSarifJSON implements common.SARIFFormat.
+// Must be used only with lint.Command because understands only run.Result as data parameter.
+func FormatSarifJSON(out io.Writer, data interface{}) error {
+	if res, ok := data.(run.Result); ok {
+		return formatSarif(out, res)
+	}
+	return errors.New("Blah") // TODO: message
+}
+
+func formatSarif(out io.Writer, result run.Result) error {
+	sarifReport, err := sarif.New(sarif.Version210)
+	if err != nil {
+		return err
+	}
+
+	// TODO: have kube-linter and github address somewhere in "more common" place for such strings.
+	sarifRun := sarif.NewRun("kube-linter", "https://github.com/stackrox/kube-linter")
```

Comment:
Is there a spot to put the `kube-linter` version in the output too?

Reply:
Added.